sealtk/core/KwiverDetectionsSink.cpp

```diff
@@ -152,10 +153,12 @@ void KwiverDetectionsSink::writeData(QUrl const& uri) const
       for (auto const& i : f.detections)
       {
         auto const& qbox = d->model->data(i, AreaLocationRole).toRectF();
+        auto const& type = d->model->data(i, ClassificationRole).toHash();
         auto kbox = kv::bounding_box_d{{qbox.left(), qbox.top()},
                                        qbox.width(), qbox.height()};
```

Comment:
This pattern has come up often enough that we may want to eventually make a dedicated `QRectF` <-> `bounding_box_d` converter class, but that is beyond the scope of this PR.

Reply:
You say that, but I *did* add `TrackUtils` in this PR. OTOH I only see two instances of this so far, and the inline code isn't unreasonable (unlike `classificationToDetectedObjectType`, which can't practically be inlined, and which is going to have a third user when type *editing* gets PR'd).